infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/CorfuReplicationManager.java

```diff
@@ -196,7 +221,7 @@ public void processStandbyChange(TopologyDescriptor newConfig) {
 
         // Remove standbys that are not in the new config
         for (String clusterId : standbysToRemove) {
-            stopLogReplicationRuntime(clusterId);
+            stopLogReplicationRuntime(clusterId, true);
```

Comment:
would it be cleaner to just call removeClusterInfoFromStatusTable() here instead?

Reply:
addressed this. Just before the last push there was logic to delete things when FSM would be stopped/restarted. After our offline discussion realized it wasn't needed.